doc/reply_by_tweet.md

```diff
@@ -0,0 +1,24 @@
+## Reply By Tweet
+
```

Comment:
Let's add a couple sentences summarizing what this all does -- i.e. `Once a minute, this system scans tweets "to:PublicLab" (or specified search, see below) for URLs matching our shortlink pattern, i.e. "https://publiclab.org/n/_____", where "_____" is a node nid. Each of the returned tweets is added as a comment to the node it responded to.`

Can you also post links to https://apps.twitter.com/ and listing the environment variables that have to be set? Thanks!